### PR TITLE
Use updated version of release-gap-package

### PR DESCRIPTION
**Pull request by zickgraf**
This PR adds the official version of release-gap-package with only minimal changes: https://github.com/zickgraf/ReleaseTools/commit/7ab0a1e0a303a22aab42204bbf27ad8fa54a9d4d

Please check if releases work fine using this version. If yes, I will try to upstream the changes in the commit linked above.

### CODE REVIEW

File: make_dist_for_homalg.sh [hunk 1]
```diff
@@ -5,7 +5,7 @@ packages="4ti2Interface ExamplesForHomalg GaussForHomalg GradedModules homalg IO
 base_dir="$PWD"
 
 for i in ${packages}; do
-  ./release --srcdir ${base_dir}/${i} --webdir ${base_dir}/gh-pages/${i} --update-file ${base_dir}/gh-pages/update.g
+  release-gap-package --no-push --srcdir ${base_dir}/${i} --webdir ${base_dir}/gh-pages/${i} --update-file ${base_dir}/gh-pages/update.g
```
Comment (mohamed-barakat) [inline make_dist_for_homalg.sh:8]:
```suggestion
  ./release-gap-package --no-push --srcdir ${base_dir}/${i} --webdir ${base_dir}/gh-pages/${i} --update-file ${base_dir}/gh-pages/update.g
```

Comment (mohamed-barakat) [reply make_dist_for_homalg.sh:8]: My path does not include "."

Comment (zickgraf) [reply make_dist_for_homalg.sh]: Should work now :-)

**Comment by mohamed-barakat** [comment]
Thank you very much :)

**Pull request merged**